pagopa/pn-frontend
# Feat/pn 386 fe pn ga visualizzazione delle notifiche ricevute persone giuridiche

## Short description
Clone personagiuridica-webapp starting from personafisica-webapp with initial change requests

## List of changes proposed in this pull request
- Sidemenu changes
- Header changes

Here's the full list of all customized (starting from pf) files:
- buildspec.yaml
- CHANGELOG.md
- package.json
- sonarqube-scanner.js
- public/locales/it/common.json
- public/locales/it/notifiche.json
- src/App.tsx
- src/navigation/routes.const.ts
- src/navigation/routes.tsx
- src/pages/Contacts.page.tsx
- src/pages/Notifiche.page.tsx
- src/pages/UnderConstruction.page.tsx

## How to test
Run the application, login using any pf user and verify verify that notifications list, notification detail and contacts sections meet the requirements
